ci/pubspec.yaml

```diff
@@ -23,15 +23,27 @@ dependencies:
   process_runner: any
   process: any
 
+dev_dependencies:
+  async_helper: any
+  expect: any
+  litetest: any
+  smith: any
+
 dependency_overrides:
   args:
     path: ../../third_party/dart/third_party/pkg/args
   async:
     path: ../../third_party/dart/third_party/pkg/async
+  async_helper:
+    path: ../../third_party/dart/pkg/async_helper
   collection:
     path: ../../third_party/dart/third_party/pkg/collection
+  expect:
+    path: ../../third_party/dart/pkg/expect
   file:
     path: ../../third_party/pkg/file/packages/file
+  litetest:
+    path: ../testing/litetest
```

Comment:
Just to double-check, are the changes in this file necessary?

Reply:
These seem to be outdated, I cleaned them up. Thanks!

Reply:
I had to re-add some of the dev dependencies because ci/test/format_test.dart uses them.